lldb/test/Shell/Commands/command-image-lookup-current-module.test

```diff
@@ -0,0 +1,43 @@
+# REQUIRES: system-darwin
```

Comment:
Wasn't sure how to create multiple modules which would work across all platforms. I'm sure there is a way, but was hoping someone to tell me :)